doc/api/fs.markdown

```diff
@@ -529,6 +532,10 @@ created. On POSIX systems, `path` is considered to exist even if it is a symlink
 to a non-existent file. The exclusive flag may or may not work with network file
 systems.
 
+`flags` can also be a number as documented by open(2); commonly used constants
+are available from `require('constants')`.  On Windows, flags are translated to
+their equivalent ones where acceptable.
+
```

Comment:
Ah seems a typo there, I wanted to say "where applicable".  Yes, it is unclear; but details imo should go to libuv docs, but no one is taking on that.  What about adding a few examples, "e.g. O_EXCL to xxx, O_RDWR to xxx..."?